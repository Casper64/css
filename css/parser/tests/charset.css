
@charset "UTF-8";

@charset 'utf-8';

@charset "ISO-12345"    ;

@charset "ASCII"

.test {}
