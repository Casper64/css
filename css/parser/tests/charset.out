css/parser/tests/charset.css:4:10: error: invalid charset rule: expecting a double quoted string after "@charset"
    2 | @charset "UTF-8";
    3 | 
    4 | @charset 'utf-8';
      |          ~~~~~~~
    5 | 
    6 | @charset "ISO-12345"    ;
css/parser/tests/charset.css:4:10: error: invalid charset rule: expecting a double quoted string after "@charset"
    2 | @charset "UTF-8";
    3 | 
    4 | @charset 'utf-8';
      |          ~~~~~~~
    5 | 
    6 | @charset "ISO-12345"    ;
css/parser/tests/charset.css:6:25: error: the ";" has to be the next character after "ISO-12345"
    4 | @charset 'utf-8';
    5 | 
    6 | @charset "ISO-12345"    ;
      |                         ^
    7 | 
    8 | @charset "ASCII"
css/parser/tests/charset.css:10:1: error: expecting a semicolon to close the at-rule
    8 | @charset "ASCII"
    9 | 
   10 | .test {}
      | ^